utilities/EluvioLiveCli.js

```diff
@@ -44,6 +44,24 @@ const Init = async ({debugLogging = false, asUrl}={}) => {
   await marketplace.Init({ debugLogging });
 };
 
+const CmdTenantAuthToken = async ({ argv }) => {
+  await Init({debugLogging: argv.verbose});
+
+  let ts = Date.now();
+  let message = argv.path_or_body + "?ts=" + ts
+  try {
+    let j = JSON.parse(argv.path_or_body)
```

Comment:
cn you share a sample url or body?  say i want to do like

```
curl -s -H 'Authorization: Bearer ES256K_...' http://localhost:6546/tnt/wlt/itenYQbgk66W1BFEqWr95xPmHZEjmdF
```
for example.

Reply:
Example output:
```sh
# URL path
$ ./elv-live tenant_auth_token "/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a"
/nft/pack/spec/0x59267d3eff5a4a595f6bfb790d18ed6af358653a?ts=1688181635434
Authorization: Bearer ES256K_6Rbkcyi7vzYzvQN4HjG3hJknUmdi1uWavxH79JFopWv6ysAQsH8c3hE7RcPpLKi4Rvxe7es8pFr1Ht52SfFvCk95C

# Request body (empty JSON payload)
$  ./elv-live tenant_auth_token "{}"
{"ts":1688181686275}
Authorization: Bearer ES256K_DAsxBZYZFcPjjK1ZajwRtCSqViRkucRHgwu7f7hQv6ENLnTEikVtcBisQqjav9p18STU5Z5rMVU3km8WXvMQB7y8v
```